syntax = "proto3";

package Protocol;

message LoginResponse{
	bool result = 1;
	string message = 2;
}